### PR TITLE
feat: add Osmosis deployments

### PR DESCRIPTION
**Pull request by byeongsu-hong**
### Description

* Added Osmosis chain definition
* Added warp route definition for TIA
* Added warp route definition for milkTIA

### CODE REVIEW

File: chains/osmosis/logo.svg
Comment (jmrossy) [inline chains/osmosis/logo.svg:1]: Please run this svg through a minifier. This is a good one you can use in your browser:
https://jakearchibald.github.io/svgomg/

**Review by jmrossy — COMMENTED** [review]
This mostly looks good! Please see my comment about the logo and please add a changeset (run `yarn changeset`)

**Comment by byeongsu-hong** [comment]
Hi @jmrossy! I've added result of running changeset, but i'm not sure if this is proper way to contribute. Can you review it?

File: deployments/warp_routes/TIA/manta-osmosis-addresses.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+neutron:
+  collateral: osmo1h4y9xjcvs8lrx4z8ha48uq9a338w74dpl2ly3tf74fzvugp2kj4q9l0jkw
+mantapacific:
+  synthetic: "0x88410F3D8135b4D23b98dC37C4652C6969a5B1a8"
```
Comment (paulbalaji) [inline deployments/warp_routes/TIA/manta-osmosis-addresses.yaml:4]:
```suggestion
mantapacific:
  synthetic: "0x88410F3D8135b4D23b98dC37C4652C6969a5B1a8"
neutron:
  collateral: osmo1h4y9xjcvs8lrx4z8ha48uq9a338w74dpl2ly3tf74fzvugp2kj4q9l0jkw
```

File: deployments/warp_routes/TIA/manta-osmosis-config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    name: TIA.o
+    addressOrDenom: "0x88410F3D8135b4D23b98dC37C4652C6969a5B1a8"
+    connections:
+      - token: cosmos|osmosis|osmo1h4y9xjcvs8lrx4z8ha48uq9a338w74dpl2ly3tf74fzvugp2kj4q9l0jkw
+options:
+  interchainFeeConstants:
+    - origin: osmosis
+      destination: mantapacific
+      amount: 840000
+      addressOrDenom: ibc/D79E7D83AB399BFFF93433E54FAA480C191248FC556924A2A8351AE2638B3877
```
Comment (paulbalaji) [inline deployments/warp_routes/TIA/manta-osmosis-config.yaml:25]:
```suggestion
# yaml-language-server: $schema=../schema.json
options:
  interchainFeeConstants:
    - addressOrDenom: ibc/D79E7D83AB399BFFF93433E54FAA480C191248FC556924A2A8351AE2638B3877
      amount: 840000
      destination: mantapacific
      origin: osmosis
tokens:
  - addressOrDenom: osmo1h4y9xjcvs8lrx4z8ha48uq9a338w74dpl2ly3tf74fzvugp2kj4q9l0jkw
    chainName: osmosis
    collateralAddressOrDenom: ibc/D79E7D83AB399BFFF93433E54FAA480C191248FC556924A2A8351AE2638B3877
    connections:
      - token: ethereum|mantapacific|0x88410F3D8135b4D23b98dC37C4652C6969a5B1a8
    decimals: 6
    name: TIA.o
    standard: CwHypCollateral
    symbol: TIA.o
  - addressOrDenom: "0x88410F3D8135b4D23b98dC37C4652C6969a5B1a8"
    chainName: mantapacific
    connections:
      - token: cosmos|osmosis|osmo1h4y9xjcvs8lrx4z8ha48uq9a338w74dpl2ly3tf74fzvugp2kj4q9l0jkw
    decimals: 6
    name: TIA.o
    standard: EvmHypSynthetic
    symbol: TIA.o
```

File: deployments/warp_routes/milkTIA/manta-osmosis-addresses.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+neutron:
+  collateral: osmo17xuecsykqw2xcxwv8cau7uy4hgdwqt0u4qxflyc2yshhggpazfjs6kfqd3
+mantapacific:
+  synthetic: "0x32474653127048d9fC20000e21dEd396b47968E8"
```
Comment (paulbalaji) [inline deployments/warp_routes/milkTIA/manta-osmosis-addresses.yaml:4]:
```suggestion
mantapacific:
  synthetic: "0x32474653127048d9fC20000e21dEd396b47968E8"
neutron:
  collateral: osmo17xuecsykqw2xcxwv8cau7uy4hgdwqt0u4qxflyc2yshhggpazfjs6kfqd3
```

File: deployments/warp_routes/milkTIA/manta-osmosis-config.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    name: milkTIA.o
+    addressOrDenom: "0x32474653127048d9fC20000e21dEd396b47968E8"
+    connections:
+      - token: cosmos|osmosis1|osmo17xuecsykqw2xcxwv8cau7uy4hgdwqt0u4qxflyc2yshhggpazfjs6kfqd3
+options:
+  interchainFeeConstants:
+    - origin: osmosis
+      destination: mantapacific
+      amount: 840000
+      addressOrDenom: factory/osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0/umilkTIA
```
Comment (paulbalaji) [inline deployments/warp_routes/milkTIA/manta-osmosis-config.yaml:25]:
```suggestion
# yaml-language-server: $schema=../schema.json
options:
  interchainFeeConstants:
    - addressOrDenom: factory/osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0/umilkTIA
      amount: 840000
      destination: mantapacific
      origin: osmosis
tokens:
  - addressOrDenom: osmo17xuecsykqw2xcxwv8cau7uy4hgdwqt0u4qxflyc2yshhggpazfjs6kfqd3
    chainName: osmosis
    collateralAddressOrDenom: factory/osmo1f5vfcph2dvfeqcqkhetwv75fda69z7e5c2dldm3kvgj23crkv6wqcn47a0/umilkTIA
    connections:
      - token: ethereum|mantapacific|0x32474653127048d9fC20000e21dEd396b47968E8
    decimals: 6
    name: milkTIA.o
    standard: CwHypCollateral
    symbol: milkTIA.o
  - addressOrDenom: "0x32474653127048d9fC20000e21dEd396b47968E8"
    chainName: mantapacific
    connections:
      - token: cosmos|osmosis1|osmo17xuecsykqw2xcxwv8cau7uy4hgdwqt0u4qxflyc2yshhggpazfjs6kfqd3
    decimals: 6
    name: milkTIA.o
    standard: EvmHypSynthetic
    symbol: milkTIA.o
```

**Review by paulbalaji — COMMENTED** [review]
lint fixes

**Pull request merged**